lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -13,6 +13,10 @@
   display: inline-block;
   position: relative;
   width: nx-dropdown-variables.$dropdown-width;
+
+  .nx-icon:not(.nx-icon--colorful):not(.nx-dropdown-link .nx-icon):not(.nx-dropdown .disabled .nx-icon) {
+    color: var(--nx-swatch-grey-05); 
```

Comment:
Semantically speaking, is it supposed to be strictly grey-05, or is it supposed to match the dark (stark) text color? I suspect it might be the latter which will make things smoother as we add dark mode support.  Check with design.

Reply:
> Semantically speaking, is it supposed to be strictly grey-05, or is it supposed to match the dark (stark) text color (which happens to be grey-05)? I suspect it might be the latter which will make things smoother as we add dark mode support. Check with design.

I asked Jenni about it, and she said all the light mode icons should be grey-05 unless in an interaction state. Since they haven't started on the icon collection for dark mode it is not sure if it will match the text color.